smtpd/filestore.go

```diff
@@ -181,6 +181,10 @@ func (mb *FileMailbox) GetMessage(id string) (Message, error) {
 		}
 	}
 
+	if id == "latest" {
+		return mb.messages[len(mb.messages)-1], nil
```

Comment:
Won't this blow up if the mailbox is empty?  Can you add that to your unit test?

Reply:
done, I overlooked that, I thought the entry point in the function checked that.